apply-pr.js

```diff
@@ -149,6 +149,10 @@ function resolveNames() {
       exitWithMsg(err);
     }
 
+    if (!name.email) {
+      exitWithMsg(util.format('Email not set for %s', name.name));
```

Comment:
can we make this a warning like, that does not exit the process, but just prints out on stderr?